### ISSUE TITLE
error [pimatic-netatmo]: Can't connect to netatmo API, retry in 30s

### ISSUE THREAD

**Issue by blacktiger333** [thexperiments/pimatic-netatmo#12]
I'm having problems to connect to netatmo. What I did:
- installed the same client id with client secret on two Raspis (my mistake), since than it wasn't possible to connect to netatmo API.
- I created for my second Raspi a new client id  - it doesn't help, my first installation (without any changes) doesn't work anymore (tried restarting pimatic and raspi)
- checked the app status on dev.netatmo.com - both apps shows Ban status No and status: OK

Any help? Is there a way to get more detailed Infos why connection to netatmo API isn't possible. Had a look on the debug logfile (with pimatic debug enabled) - but didn't find more Infos.

**Comment by thexperiments** [comment]
What is the log output? I was using netatmo API from my live system and from my dev system at the same time and didn't have any problems.

**Comment by blacktiger333** [comment]
As I wrote:

17:52:07.786 [pimatic-netatmo] error: Can't connect to netatmo API, retry in 30s

17:52:42.886 [pimatic-netatmo] error: Can't connect to netatmo API, retry in 30s

17:53:13.004 [pimatic-netatmo] error: Can't connect to netatmo API, retry in 30s

17:53:43.116 [pimatic-netatmo] error: Can't connect to netatmo API, retry in 30s

17:54:14.016 [pimatic-netatmo] error: Can't connect to netatmo API, retry in 30s

17:54:44.152 [pimatic-netatmo] error: Can't connect to netatmo API, retry in 30s

17:55:14.292 [pimatic-netatmo] error: Can't connect to netatmo API, retry in 30s

17:55:44.415 [pimatic-netatmo] error: Can't connect to netatmo API, retry in 30s

17:56:14.534 [pimatic-netatmo] error: Can't connect to netatmo API, retry in 30s

17:56:44.647 [pimatic-netatmo] error: Can't connect to netatmo API, retry in 30s



No further information in pimatic-deamon.log. Debugging is enabled.



Von: thexperiments [mailto:notifications@github.com]
Gesendet: Sonntag, 11. Dezember 2016 17:55
An: thexperiments/pimatic-netatmo <pimatic-netatmo@noreply.github.com>
Cc: blacktiger333 <chris3chris3chris@hotmail.com>; Author <author@noreply.github.com>
Betreff: Re: [thexperiments/pimatic-netatmo] error [pimatic-netatmo]: Can't connect to netatmo API, retry in 30s (#12)



What is the log output? I was using netatmo API from my live system and from my dev system at the same time and didn't have any problems.

**Comment by thexperiments** [comment]
looks like a general networ issue. The plugin prints this if there is no response from the API.

Can you ping api.netatmo.net from the raspberry?

Maybe your IP is banned for some reason? Try reconnecting your router to get a new IP from your provider.

**Comment by blacktiger333** [comment]
Yes – I can sudo ping api.netatmo.net without problems from the raspi. I’m having a static IP from my provider – but I use also other apps from my LAN (Netatwin under Windows 10) with my public IP address to access netatmo – without any problems, so I assume my IP isn’t banned. Very strange.



Von: thexperiments [mailto:notifications@github.com]
Gesendet: Sonntag, 11. Dezember 2016 18:11
An: thexperiments/pimatic-netatmo <pimatic-netatmo@noreply.github.com>
Cc: blacktiger333 <chris3chris3chris@hotmail.com>; Author <author@noreply.github.com>
Betreff: Re: [thexperiments/pimatic-netatmo] error [pimatic-netatmo]: Can't connect to netatmo API, retry in 30s (#12)



looks like a general networ issue. The plugin prints this if there is no response from the API.

Can you ping api.netatmo.net from the raspberry?

Maybe your IP is banned for some reason? Try reconnecting your router to get a new IP from your provider.

**Comment by blacktiger333** [comment]
Tried the api via web with my public ip address from my pc – no problems so far, so I assume – it must be a raspi pimatic problem. Also, as I wrote – I can reach api.netatmo.net from raspi with sudo ping. My credentials shouldn’t be locked, because I used the same access_token what I’m using with my pimatic-netatmo config. Also restarted my raspi – doesn’t help. What is the URL you are using to get connected and read the data? I want to try this URL from my PC – and if successfully from RASPI GUI.









Von: thexperiments [mailto:notifications@github.com]
Gesendet: Sonntag, 11. Dezember 2016 18:11
An: thexperiments/pimatic-netatmo <pimatic-netatmo@noreply.github.com>
Cc: blacktiger333 <chris3chris3chris@hotmail.com>; Author <author@noreply.github.com>
Betreff: Re: [thexperiments/pimatic-netatmo] error [pimatic-netatmo]: Can't connect to netatmo API, retry in 30s (#12)



looks like a general networ issue. The plugin prints this if there is no response from the API.

Can you ping api.netatmo.net from the raspberry?

Maybe your IP is banned for some reason? Try reconnecting your router to get a new IP from your provider.

**Comment by blacktiger333** [comment]
Problem solved. It was the Netatmo api which seems to be locked (or any other problem). After deactivating the plugin for 24h it works now.

Just a question regarding the default interval for measuring. Is the default value 60 in seconds? I ask for it - because I left the default interval value for my netatmo devices and it measures every 10 minutes.
